### DISCUSSION TITLE
Streams ducking audio when highlighted team radio plays

### DISCUSSION THREAD

**Discussion by ttenpi** [f1multiviewer/issue-tracker#516] [Feature requests]
(Similar to #200, but for the on-demand team radio highlights instead.)

I would like to see an option (either settings, or cogwheel on the bottom of Live Timing windows) to duck all open streams whenever a team radio is selected and plays. I would also like to be able to control by how much (% in volume) it would be ducked.

**Comment by jeppeolesen** [comment]
Marked as a duplicate of #359
